checkstyle/java.header

```diff
@@ -1,5 +1,5 @@
 /\*
- \* Copyright 20(19|2[0-9]) Aiven Oy
+ \* Copyright (20(19|2[0-9]) - )?20(19|2[0-9]) Aiven Oy and jdbc-connector-for-apache-kafka project contributors
```

Comment:
review note:

This allows:

- Single year
  - `2019`
  - `2020` to `2029`
- Ranges of the same values as above, with the separator ` - `, e.g.:
  - `2019 - 2019`
  - `2020 - 2019`
  - `2019 - 2022`